### PR TITLE
add bootstrap and navigation bar

### PR DESCRIPTION
**Pull request by zoebarth**
Add bootstrap and placeholder navigation bar.

Add bootstrap to make site more aesthetically pleasing, and change the width of the page in css. Add placeholder navigation bar to show the structure of the site when completed.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,13 +1,27 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (mecolmg) [inline portfolio/src/main/webapp/index.html:2]: Nice addition. Minor nit: could be en-US

**Comment by zoebarth** [comment]
Screenshot 2020-05-28 at 11 48 22 AM

**Comment by zoebarth** [comment]
Screenshot 2020-05-28 at 11 50 30 AM

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (zoebarth) [reply portfolio/src/main/webapp/index.html]: fixed it

**Pull request merged**